### ISSUE TITLE
itunes

### ISSUE THREAD

**Issue by PANOSSS1980** [moonheekim0118/pythonplayground_itunes#1]
MY FRIEND HELLO!!!TELL ME PLEASE THIS IS FOR SEARCHING MUSIC SONGS AND ALBUMS FROM ITUNES?

**Comment by moonheekim0118** [comment]
No, it's just a program that analyzes iTunes music's play time and rating :( it's from book called python playground.
